packages/frontend/core/src/components/workspace/index.css.ts

```diff
@@ -52,12 +52,16 @@ export const mainContainerStyle = style({
   flex: 1,
   overflow: 'hidden',
   maxWidth: '100%',
+  transition: 'margin-left 0.2s ease',
   selectors: {
+    '&[data-client-border="true"]': {
+      borderRadius: 6,
+    },
     '&[data-show-padding="true"]': {
       margin: '8px',
       overflow: 'hidden',
-      // todo: is this performance intensive?
-      filter: 'drop-shadow(0px 0px 4px rgba(66,65,73,.14))',
+      // TODO: not match with design's shadow, theme missing
+      boxShadow: '0px 0px 4px rgba(66,65,73,.14)',
```

Comment:
Using box-shadow has a small glitch

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/102c3b11-1dd7-447d-b093-c082a3d8f0e6.png)

Reply:
Could you elaborate?

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/4e71cc55-d868-42fb-8e3c-e744a2a4f8b0.png)

you will see it when zoom in

Reply:
Got it! Using `drop-shadow` is really wise!

Reply:
I guess we can also move shadows to individual frames to get rid of using cpu consuming filters

Reply:
Not sure how much benefit there will be, but it would require additional handling of `box-shadow` in the sidebar and split-view.